src/Silverback.Integration.Kafka/Messaging/Configuration/Kafka/ConfluentClientConfigProxies.cs

```diff
@@ -926,7 +942,7 @@ public string ConsumeResultFields
         public string GroupId
         {
             get => ConfluentConfig.GroupId;
-            set => ConfluentConfig.GroupId = value;
+            set => ConfluentConfig.GroupId = string.IsNullOrWhiteSpace(value) ? GroupIdNotSet : value;
```

Comment:
If the value is never set, how does it get the unset value?

Reply:
Has been moved to:
https://github.com/BEagle1984/silverback/blob/491926231cbcd0b64afe9452e05c97e0fe55ff05/src/Silverback.Integration.Kafka/Messaging/Configuration/Kafka/KafkaConsumerConfig.cs#L56